### PR TITLE
Nazanin/date_of_birth

### PR DESCRIPTION
**Pull request by nazaninreihani**

### CODE REVIEW

File: src/javascript/app/pages/user/new_account/financial_acc_opening.js [hunk 1]
```diff
@@ -42,16 +42,16 @@ const FinancialAccOpening = (() => {
             Object.keys(get_settings).forEach((key) => {
                 $element = $(`#${key}`);
                 value    = get_settings[key];
-                if (key === 'date_of_birth') {
+                if (key === 'date_of_birth' && value) {
                     const moment_val = moment.utc(value * 1000);
                     get_settings[key] = moment_val.format('DD MMM, YYYY');
                     $element.attr({
                         'data-value': toISOFormat(moment_val),
+                        'value'     : toISOFormat(moment_val),
```
Comment (kellybinary) [inline src/javascript/app/pages/user/new_account/financial_acc_opening.js]: should this be converted to readable format instead?

Comment (nazaninreihani) [reply src/javascript/app/pages/user/new_account/financial_acc_opening.js]: @negarn @kellybinary What do you suggest?

Comment (negarn) [reply src/javascript/app/pages/user/new_account/financial_acc_opening.js]: in desktop yes, but I believe in mobile there's no way for that.

Comment (kellybinary) [reply src/javascript/app/pages/user/new_account/financial_acc_opening.js]: it seems that the helper file `date_picker.js` will handle the value to be displayed, i think it'll be fine if you remove it

Comment (nazaninreihani) [reply src/javascript/app/pages/user/new_account/financial_acc_opening.js]: I added it to fix the issue when you load the page in mobile view the date of birth is empty although it has value

**Pull request merged**